tests/integration/components/o-s-s/search-field-test.ts

```diff
@@ -0,0 +1,66 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, find, click, typeIn } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupIntl } from 'ember-intl/test-support';
+
+module('Integration | Component | o-s-s/search-field', function (hooks) {
+  setupRenderingTest(hooks);
+  setupIntl(hooks);
+
+  test('Search Field component is rendered with a placeholder', async function (assert) {
+    this.set('placeholder', 'This is a placeholder');
+    this.set('value', '');
+    this.set('onChange', (value: string) => {
+      this.value = value;
+    });
+
+    await render(
+      hbs`<OSS::SearchField @value={{this.value}} @placeholder={{this.placeholder}} @onChange={{this.onChange}} />`
+    );
+
+    assert.strictEqual(
+      find('input')?.getAttribute('placeholder'),
+      'This is a placeholder',
+      'The placeholder is correctly set.'
+    );
```

Comment:
Although this works fine, the `assert` module actually have a shorter way to do this:

```
assert.dom('input').hasAttribute('placeholder', 'This is a placeholder');
```

You can the see the documentation [here](https://github.com/mainmatter/qunit-dom/blob/master/API.md#hasattribute) along with some other functions that exist :)